dot4gravity/src/lib.rs

```diff
@@ -456,7 +456,7 @@ impl<Player: PartialEq + Clone> Game<Player> {
         mut game_state: GameState<Player>,
         player: Player,
         side: Side,
-        position: u8,
+        position: Position,
     ) -> Result<GameState<Player>, GameError> {
         Self::can_drop_stone(&game_state, &side, position, &player)?;
         let player_index = game_state.player_index(&player);
```

Comment:
could it be clearer to define the `usize` type we use here instead of within the array lookups below?